### ISSUE TITLE
[DOCUMENTATION]: Icons - SHOULD have updated documentation for better accessibility

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#10434]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Our icons should have updated documentation for when to use `role="img"` and when to use `role="presentation"`. There are times both are appropriate, and it depends on how the icon is used. We should also consider adding a default role to these icons.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] Documentation is updated to highlight when developers should use different roles
- [ ] If an icon is being used alone (without support text), it should have a `role="img"` AND appropriate ARIA markup OR screen reader only descriptive text
- [ ] If an icon is being used with visual (and assistive device) text, it should have `role="presentation"`
- [ ] Documentation should show more explicit use cases:
  * Icon being used alone
  * Icon being used with supporting text
  * Icon being used alone inside an `<a>` tag
  * icon being used with supporting text inside an `<a>` tag

## Environment

* https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/iconbase/
* https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/iconhelp/
* https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/iconsearch/
* https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/visual-design/components/iconuser/

## WCAG or Vendor Guidance (optional)
* https://developer.paciellogroup.com/blog/2013/12/using-aria-enhance-svg-accessibility/

**Comment by 1Copenut** [comment]
Issue moved to [department-of-veterans-affairs/vets-design-system-documentation #221](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/221) via [**ZenHub**](https://www.zenhub.com/)
